src/config.rs

```diff
@@ -2271,6 +2273,7 @@ impl Default for CdcConfig {
             old_value_cache_memory_quota: ReadableSize::mb(512),
             // Deprecated! preserved for compatibility check.
             old_value_cache_size: 0,
+            bandwidth_limit: ReadableSize::mb(512),
```

Comment:
```suggestion
            network_bandwidth_limit: ReadableSize::mb(128),
```

I think 128MB/s is large enough given that the default scan speed is 128MB/s too.